Assets/Scripts/Ship Movement/ShipMovement.cs

```diff
@@ -41,7 +42,6 @@ public class ShipMovement : MonoBehaviour
     void Start()
     {
         Instance = this;
-        Cursor.lockState = CursorLockMode.Locked; //keep mouse in the game
```

Comment:
Add this back in! At the very start of the game, when main scene is first loaded, the mouse is not locked.